packages/api-plugin-fulfillment/src/mutations/updateFulfillmentType.js

```diff
@@ -0,0 +1,60 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+const inputSchema = new SimpleSchema({
+  fulfillmentGroupId: String,
+  shopId: String,
+  name: String,
+  enabled: {
+    type: Boolean,
+    defaultValue: true
+  },
+  label: String,
+  displayMessageType: {
+    type: String,
+    optional: true
+  }
```

Comment:
Usually for an `update*` mutation, all of these would be optional except `fulfillmentGroupId` and `shopId`, and if they are not passed in they would not be updated.

Reply:
Updated to make all fields except `fulfillmentTypeId` and `shopId` optional.
Will check for each optional field if there is input data and then accordingly create the `updateObject` to be passed to `updateOne`